students/aminavi/session04/trigram3.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3 
+import random
+# path C:\\Users\\AMinavi1\\Au2018-Py210B\\students\\aminavi\\session04\\
+# file name pg20151.txt
+
+def read_file():
+    # fp = input("please enter the path ")
+    # fn = input("please enter the file name ")
+    fp = 'C:\\Users\\AMinavi1\\Au2018-Py210B\\students\\aminavi\\session04\\'
```

Comment:
We talked about this in class, and this was a quick workaround to move forward.

It's always strongly encouraged that your data files are stored under the current directory by using a relative path. I suppose you were finding your files at your `C:\Users\Aminavi1\Au2018-Py210B`, not under `students\aminavi\session04` as desired, so you hard-coded the whole thing. In that case, you can actually start a new VSCode window, and open your `'C:\\Users\\AMinavi1\\Au2018-Py210B\\students\\aminavi\\session04\\'` folder, which will create your `.txt` files in your `session04` subdirectory...